api/build.go

```diff
@@ -1150,7 +1151,10 @@ func CancelBuild(c *gin.Context) {
 				util.HandleError(c, http.StatusBadRequest, retErr)
 				return
 			}
-			c.JSON(resp.StatusCode, strings.Trim(string(respBody), "\""))
+
+			json.Unmarshal(respBody, b)
```

Comment:
can we check the err from Unmarshal here?
```suggestion
			err = json.Unmarshal(respBody, b)
			if err != nil {
			
		            util.HandleError(c, http.StatusInternalServerError, err)

		            return
	                }
			
```

Reply:
TYVM 😄